src/components/skeletons/skeletons.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+
+import classes from "./skeletons.module.css";
+
+const SkeletonTextLine: React.FC<{
+  fontSize: 1.4 | 1.6;
+  width: 40 | 50 | 75;
+  responsive?: boolean;
+}> = ({ fontSize, width, responsive }) => {
+  let dynamicClassNames;
+
+  if (fontSize === 1.4) {
+    if (width === 40) {
+      dynamicClassNames = "h-6 w-2/5";
+    } else if (width === 50) {
+      dynamicClassNames = "h-6 w-1/2";
+    } else if (width === 75) {
+      dynamicClassNames = "h-6 w-3/4";
+    }
+  } else if (fontSize === 1.6) {
+    if (width === 40) {
+      dynamicClassNames = "h-7 w-2/5";
+    } else if (width === 50) {
+      dynamicClassNames = `h-7 ${responsive ? "w-3/4 lg:w-1/2" : "w-1/2"}`;
+    } else if (width === 75) {
+      dynamicClassNames = "h-7 w-3/4";
+    }
+  }
+
+  return (
+    <div
+      className={`relative bg-box rounded overflow-hidden ${dynamicClassNames} ${classes.skeletonAnimation}`}
+    />
+  );
+};
```

Comment:
Several things here:

First, before migration to TW, we used the font size of the awaited text to define directly the height of the skeleton text line, to avoid unwanted jumps of content. I had to adapt this a bit, since 1.4rem & 1.6rem (14 & 16px) doesn't exists in TW height property, so I used 1.5rem for the first and 1.75rem for the second. It seems not noticeable from what I've seen.
Second, to be able to adapt the component on every case, I had to do this bunch of if...else if to cover all cases with TW. It is the less ugly way I found to do it.
Last thing, I added an argument responsive to determine if the component needs adaption on smaller screens. The skeleton text line in mobile for identities was a bit too small, so with this new argument, I augmented them to 75% instead of usual 50% width on mobile and preserved 50% in desktop. For now, it is used just when the value of width is 50, so I didn't implemented it in other case.